src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/config.html.twig

```diff
@@ -126,74 +108,53 @@
 {% endblock %}
 
 {% block panel %}
-    {% if collector.applicationname %}
-        {# this application is not the Symfony framework #}
```

Comment:
i actually saw

![image](https://user-images.githubusercontent.com/1047696/47264817-e5bdf700-d51d-11e8-901a-1f80e7a98f65.png)


 :|

Reply:
I think we can remove both application name and application version. It was used by Silex, but not needed anymore. Let's simplify.

Reply:
you mean keep the feature, and only remove the wired argument? is it really something we want to support though? I've seen this for the first now.

Reply:
ok. it's ripped out fully now :) should the constructor trigger a deprecation if passed?

Reply:
@fabpot actually it was used also by our apps to show app's name and version (from tag or from branch's name). Too bad it was removed.